lib/serverspec/type/php_extension.rb

```diff
@@ -0,0 +1,11 @@
+module Serverspec::Type
+  class PhpExtension < Base
+    def loaded?
+      ret = @runner.run_command("php --ri '#{@name}'")
+
+      return false if ret.exit_status.to_i > 0
```

Comment:
Made this `> 0` to account for whatever other non-zero exit code there could be.